spdx/v2/v2_3/tagvalue/reader/parse_creation_info_test.go

```diff
@@ -425,3 +425,14 @@ func TestFailsExternalDocumentReferenceWithInvalidFormats(t *testing.T) {
 		}
 	}
 }
+func TestParserCIMovesToOtherLicenseAfterParsingDocument(t *testing.T) {
```

Comment:
This appears to have the same naming issue as the other PR, and I don't see a change updating the other one

Reply:
Fixed, thanks!